CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
+ * `.ent` and `.mod` files should be parsed as DTDs. See[#380](https://github.com/angelozerr/lsp4xml/issues/380)
+ * Add support for `textDocument/typeDefinition` from XML to XMLSchema/DTD . See[#371](https://github.com/angelozerr/lsp4xml/issues/371)
+ * Show possible elements for completion based on XML Schema. See[#347](https://github.com/angelozerr/lsp4xml/issues/347)
+ * Add support for `textDocument/definition` for XML Schema. See[#148](https://github.com/angelozerr/lsp4xml/issues/148)
+ * Add support for `textDocument/references` for XML Schema types. See[#58](https://github.com/angelozerr/lsp4xml/issues/58)
+ * Add support for `textDocument/codelens` for XML Schema types. See[#55](https://github.com/angelozerr/lsp4xml/issues/55)
+ * Add completion for `comment` and `#region`. See[#54](https://github.com/angelozerr/lsp4xml/issues/54)
+ * CData block auto-complete. See[#168](https://github.com/redhat-developer/vscode-xml/issues/168)
+ * Documentation in completion has monospace font. See[#32](https://github.com/redhat-developer/vscode-xml/issues/32)
+
+### Bug Fixes
+
+ * Hover markup kind response ignores the hover client capability. See[#525](https://github.com/angelozerr/lsp4xml/issues/525)
+ * Loss of completion capability from the client. See[#522](https://github.com/angelozerr/lsp4xml/issues/522)
+ * Error range for `EntityNotDeclared`. See[#518](https://github.com/angelozerr/lsp4xml/issues/518)
+ * Improve revalidate XML files when XSD files is saved. See[#506](https://github.com/angelozerr/lsp4xml/issues/506)
+ * Error range for `src-import.1.2`. See[#499](https://github.com/angelozerr/lsp4xml/issues/499)
+ * Error range for `s4s-elt-invalid-content.3`. See[#496](https://github.com/angelozerr/lsp4xml/issues/496)
+ * NPE in `textDocument/definition` in XSD files. See[#488](https://github.com/angelozerr/lsp4xml/issues/488)
+ * Error range for `cvc-pattern-valid`. See[#477](https://github.com/angelozerr/lsp4xml/issues/477)
+ * Error range for `AttributePrefixUnbound`. See[#476](https://github.com/angelozerr/lsp4xml/issues/476)
+ * Error range for `EmptyTargetNamespace`. See[#472](https://github.com/angelozerr/lsp4xml/issues/472)
+ * Error range for `ct-props-correct.3`. See[#467](https://github.com/angelozerr/lsp4xml/issues/467)
+ * Error range for `sch-props-correct.2`. See[#462](https://github.com/angelozerr/lsp4xml/issues/462)
+ * Error range for `s4s-elt-must-match.2`. See[#458](https://github.com/angelozerr/lsp4xml/issues/458)
+ * Error range for `ct-props-correct.3`. See[#455](https://github.com/angelozerr/lsp4xml/issues/455)
+ * Error range for `src-ct.1`. See[#453](https://github.com/angelozerr/lsp4xml/issues/453)
+ * Error range for `duplicate attribute`.. See[#452](https://github.com/angelozerr/lsp4xml/issues/452)
+ * Error range for `p-props-correct.2.1`. See[#436](https://github.com/angelozerr/lsp4xml/issues/436)
+ * Element/Attribute names must be case sensitive. See[#433](https://github.com/angelozerr/lsp4xml/issues/433)
```

Comment:
Fixed case sensitivity problems for element and attribute names

Reply:
@xorye updated